corehq/apps/dashboard/templates/dashboard/base.html

```diff
@@ -206,5 +206,22 @@ <h1>
       </p>
     </div>
   {% endif %}
+  {% if request|ui_notify_enabled:"APP_USER_PROFILES_UPDATE" %}
+      <div class="alert alert-ui-notify alert-ui-notify-relative alert-dismissible helpbubble helpbubble-purple helpbubble-top-left fade in"
+           style="position: absolute; width: 200px;"
+           data-target="#ProjectUsersTab"
+           data-slug="{{ "APP_USER_PROFILES_UPDATE"|ui_notify_slug }}"
+           role="alert">
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
The aria-label should get translated, too. There's a `trans_html_attr` template tag to use for attributes, which handles escaping quotes.